pyproject.toml

```diff
@@ -274,6 +274,7 @@ warn_return_any = true
 warn_unused_configs = true
 disable_error_code = "import-untyped" # do not check imports
 exclude = ["build"]
+python_version = "3.10"
```

Comment:
Does mypy default to something <3.10 without this line? We'll have to keep this line in mind wrt. the upcoming drop of 3.10 support.

Reply:
@tapastro is that ok with you? if so I will go ahead and merge

Reply:
Yeah, that's fine. Merge away 🫡